sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/AbstractScheduler.java

```diff
@@ -403,6 +391,18 @@ private void processQueuedOffers() {
                 return;
             }
 
+            // Task Reconciliation:
```

Comment:
It _looks_ like this is just a repeat of #1759 as opposed to a regression from it so I'll just ignore it

Reply:
Yes, this was based on that branch which is now merged.